Patches/DefensiveMachineGunTurretPack/ThingDefs/DMGT_Defensive_Turrets.xml

```diff
@@ -162,17 +180,16 @@
 						<canTargetLocations>true</canTargetLocations>
 					</targetParams>
 					<muzzleFlashScale>10</muzzleFlashScale>
-					<recoilPattern>Mounted</recoilPattern>
```

Comment:
recoilPattern removed?

Reply:
I was under the assumption that CE firearms no longer needed `recoilPattern`. Or do turrets still use these?

Reply:
All I can tell you for certain is that _all_ of the turrets within CE still use `<recoilPattern>Mounted</recoilPattern>`, as does the LMG (which I presume is to essentially mean it's being fired from a bipod). Now, that doesn't guarantee it still _does anything,_ but it is still there.

As for other firearms, I can only assume there's some other, default value it defaults to when not specified that covers basically all other firearms.

Reply:
Fixed in 1f3f441